app/models/site.rb

```diff
@@ -1,4 +1,19 @@
 class Site < Base
   has_one :site_status
-  belongs_to :provider
+  belongs_to :provider, param: :provider_code
+
+  REGIONS = [
+    ['London', :london],
+    ['South East', :south_east],
+    ['South West', :south_west],
+    ['Wales', :wales],
+    ['West Midlands', :west_midlands],
+    ['East Midlands', :east_midlands],
+    ['Eastern', :eastern],
+    ['Noth West', :north_west],
+    ['Yorkshire and the Humber', :yorkshire_and_the_humber],
+    ['North East', :north_east],
+    ['Scotland', :scotland],
+    ['No Region', :no_region],
+  ].freeze
```

Comment:
Maybe what we should do is be requesting this list from the API once and cache it. There might be a few things we want to query the backend about on startup ...

Reply:
(to be clear, not worth doing now, just something to keep in mind)